tools/mksprite/convtool.c

```diff
@@ -13,25 +13,31 @@ int main( int argc, char *argv[] )
     uint8_t zero = 0;
     uint8_t val;
 
+    #ifndef _MSC_VER
+    int ret __attribute__ ((unused));
+    #else
+    int ret;
+    #endif
+
     /* Read in old width */
-    fread( &val, sizeof( val ), 1, stdin );
+    ret = fread( &val, sizeof( val ), 1, stdin );
```

Comment:
If we are assigning ret, would be nice to use it instead of doing it only to make compiler happy (if that's the case here)